packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js

```diff
@@ -43,27 +45,19 @@ export default class AgoraIpOrderCtrl {
     this.IpOrganisation = IpOrganisation;
     this.User = User;
     this.atInternet = atInternet;
+    this.IP_AGORA = IP_AGORA;
   }
 
   $onInit() {
     this.model = {
       params: {},
       selectedService: null,
     };
-
+    this.loadingPage = true;
```

Comment:
I don't see the use of this variable, if we are not using, can we remove it?

Reply:
yes will remove it.